curated_transformers/tests/models/gpt_neox/test_causal_lm.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import torch
+
+from curated_transformers._compat import has_hf_transformers, transformers
+from curated_transformers.models.gpt_neox.causal_lm import GPTNeoXCausalLM
+from curated_transformers.tests.util import torch_assertclose
+
+
+@pytest.mark.slow
```

Comment:
We can remove the slow markers now, I think?

Reply:
Done.